### PR TITLE
Extracting prospero's name

### PR DESCRIPTION
**Pull request by spyrkob**
There's a couple more things to do - stopping/starting service when updating, using RHN to download prospero, but I'd rather do those in separate PRs.

@rpelisse could you review please?

### CODE REVIEW

File: roles/wildfly_prospero/tasks/main.yml [hunk 1]
```diff
@@ -54,11 +57,33 @@
...
+      ansible.builtin.set_fact:
+        prospero_command_args: "{{ prospero_command_args }} --repositories {{ eap_prospero_override_repos }}"
+      when:
+        - eap_prospero_override_repos is defined
+    - name: Override provisioning manifest
+      ansible.builtin.set_fact:
+        prospero_command_args: "{{ prospero_command_args }} --manifest {{ wildfly_install_manifest_dir }}/wildfly-{{ wildfly_version }}-manifest.yaml"
+      when:
+        - not eap_enable is defined or not eap_enable
+    - name: Display prospero command arguments
```
Comment (rpelisse) [inline roles/wildfly_prospero/tasks/main.yml:79]: @spyrkob I would keep the debug stmt but add a 

```
when: 
  - wildfly_prospero_debug_enabled is defined and wildfly_prospero_debug_enabled
```

**Review by rpelisse — CHANGES_REQUESTED** [review]
@spyrkob Just a tweak to sugget, otherwise, LGTM!

File: roles/wildfly_prospero/tasks/main.yml (hunk 1 above)
Comment (spyrkob) [reply roles/wildfly_prospero/tasks/main.yml:79]: Done, thanks!

**Review by rpelisse — COMMENTED** [review]
LGTM

**Review by rpelisse — APPROVED** [review]

**Pull request merged**